feature-staking-impl/src/main/java/io/novafoundation/nova/feature_staking_impl/presentation/validators/change/custom/review/ReviewValidatorsRouter.kt

```diff
@@ -0,0 +1,26 @@
+package io.novafoundation.nova.feature_staking_impl.presentation.validators.change.custom.review
+
+import io.novafoundation.nova.feature_staking_impl.presentation.StakingRouter
+
+interface ReviewValidatorsRouter {
+
+    fun openNextScreen()
+}
```

Comment:
I suggest to merge it with `ReviewValidatorsFlowAction`. Imho they should be a single entity. It will allow us to switch on FlowType only once